Sources/TensorFlow/Core/LazyTensorContext.swift

```diff
@@ -62,12 +62,22 @@ class LazyTensorOperationsTracker {
 
 struct LazyTensorContext {
     private var operationsTracker = LazyTensorOperationsTracker()
+    private var _shapeTrackingEnabled = true
```

Comment:
```suggestion
    private var isShapeTrackingEnabled = true
```

No need for underscore in a less-than-public type.